src/components/list-redirections.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { RedirectionProps, Dispatch } from '../lib/redirects-manager-state';
+import { sortByMultipleProperties } from '../lib/utils';
+
+import { Redirection } from './redirection';
+import { Paginator } from './paginator';
+
+type getViewType = (
+    store: RedirectionProps[],
```

Comment:
can we please make this naming more specific. "store" may be confused with a full blown store-object, but in reality, It's only a list of redirects. Better call it `items` or `redirects`, because that's what it is.

Reply:
applied